### PR TITLE
Adding EVScore: the expected value score

### PR DESCRIPTION
**Pull request by maxhutch**
# Citrine Python PR

## Description 
Wrote and tested the wrapper and updated the docs.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [x] I have added tests for 100% coverage
- [x] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [x] I have bumped the version in setup.py

### CODE REVIEW

File: tests/informatics/test_scores.py [hunk 1]
```diff
@@ -37,6 +37,21 @@ def mei_score() -> EIScore:
     )
 
 
+@pytest.fixture
+def mev_score() -> EVScore:
+    """Build an MEVScore."""
```
Comment (bfolie) [inline tests/informatics/test_scores.py:42]: Let's use "ev" instead of "mev" here

File: tests/informatics/test_scores.py [hunk 2]
```diff
@@ -55,3 +70,13 @@ def test_mei_initialization(mei_score):
     assert mei_score.baselines == [1.0]
     assert isinstance(mei_score.constraints[0], ScalarRangeConstraint)
     assert mei_score.constraints[0].descriptor_key == 'y'
+
+
+def test_mev_initialization(mev_score):
```
Comment (bfolie) [inline tests/informatics/test_scores.py:75]: same "mev" -> "ev"

Comment (maxhutch) [reply tests/informatics/test_scores.py:75]: https://github.com/CitrineInformatics/citrine-python/pull/281

File: tests/informatics/test_scores.py (hunk 1 above)
Comment (maxhutch) [reply tests/informatics/test_scores.py:42]: https://github.com/CitrineInformatics/citrine-python/pull/281

**Pull request merged**